packages/ui/src/services/color-config/color-functions.ts

```diff
@@ -6,10 +6,13 @@ export const colorToRgba = (color: ColorInput, opacity: number) => {
   return new ColorTranslator(color).setA(opacity).RGBA
 }
 
-export const isLightBackground = (color: ColorInput, opacity = 1) => {
-  // TODO: replace with color mixin (doesn't play any role in dark theme)
+export const isLightBackground = (color: ColorInput, opacity = 0.6, edge = 120) => {
```

Comment:
Do not use opacity here at all. Better use applyColors(actuallBg, bg)

Opacity 0.6 doesn't mean that is it light bg...